chain/actors/version.go

```diff
@@ -0,0 +1,24 @@
+package actors
+
+import (
+	"fmt"
+
+	"github.com/filecoin-project/go-state-types/network"
+)
+
+type Version int
+
+const (
+	Version0 = iota
+	Version1
+)
+
+// VersionForNetwork resolves the network version into an specs-actors version.
+func VersionForNetwork(v network.Version) Version {
+	switch v {
+	case network.Version0, network.Version1:
+		return Version0
+	default:
+		panic(fmt.Sprintf("unimplemented network version: %d", v))
```

Comment:
Could also return an error, but this is likely a programmer error.

Reply:
This looks duplicated from `chain/actors/builtin/builtin.go` (which is probably a slightly better place for it).

Reply:
(I was looking for that...)